crates/app/tests/mod.rs

```diff
@@ -89,12 +95,117 @@ async fn app() {
     )
     .await;
 
+    println!("Put testing file in foo tag");
+
     let (test_resp, test_type) =
         tree::get_path(&cl, &addr, &foo, &tag, &"/".parse().unwrap()).await;
 
     assert_eq!(test_type, mime::TEXT_PLAIN);
     assert_eq!(&test_resp[..], b"testing");
 
+    let random_bytes = [
```

Comment:
This is unmaintainable, what is this variable for? Where did these values come from?
Apart from that `u8` is redundant

If you need random bytes, generate them at runtime.
If you need a constant, use a human-readable string `const`.